src/components/AccountSettings.tsx

```diff
@@ -61,17 +68,38 @@ const AccountSettings = (props: AccountSettingsProps) => {
                     mask,
                     accessToken
                 )
-
                 setUserProfile(bundle.userProfile)
                 setOrcid(bundle.ORCID)
                 setVerfied(bundle.isVerified)
+                setverificationSubmission(bundle.verificationSubmission)
+                setVerificationState(bundle.verificationSubmission?.stateHistory?.slice(-1)[0])
+
             } catch (err: any) {
                 displayToast(err.reason as string, 'danger')
             }
         }
         getData()
     }, [accessToken])
 
+    const ProfileValidationState = (verificationStateEnum: VerificationStateEnum | undefined) => {
+        const profileValidationStatus = () => {
+            if(verificationStateEnum === 'SUBMITTED'){
+                return <><img className="verifyBadgeIcon" src={CheckmarkBadgeDark} alt='CheckmarkBadgeDark'/>Pending Verification</>
+            } else if(verificationStateEnum === 'REJECTED' || verificationStateEnum ==='SUSPENDED'){
+                return (
+                    <div style={{border:'solid',padding:'24px', borderColor:'#4DB3B9'}}>
```

Comment:
Please replace inline styles with css class names (defined in the Account Settings scss)